tests/rbac/test_middleware.py

```
@@ -1316,6 +1316,93 @@ def test_bootstraps_tenants_if_user_id_is_missing(self, _):
+
+            # Verify a Principal was created for the new user
+            principal = Principal.objects.get(username="brand_new_admin", tenant=tenant)
+            self.assertEqual(principal.user_id, "u78")
+
+            # Verify group membership tuples were created
+            mapping = TenantMapping.objects.get(tenant=tenant)
+            default_group_tuple_count = self._tuples.count_tuples(
+                all_of(
+                    resource("rbac", "group", str(mapping.default_group_uuid)),
+                    relation("member"),
+                    subject("rbac", "principal", f"redhat/u78"),
```

Ruff (0.16.3)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)